modules/http/http_filters.c

```diff
@@ -254,21 +254,28 @@ static apr_status_t parse_chunk_size(http_ctx_t *ctx, const char *buffer,
 }
 
 static apr_status_t read_chunked_trailers(http_ctx_t *ctx, ap_filter_t *f,
-                                          apr_bucket_brigade *b, int merge)
+                                          apr_bucket_brigade *b)
```

Comment:
Read HTTP/1.1 end chunk and append it as a HEADERS bucket to the brigade (if indeed trailers were present).